examples_apps/run_tutorial_similarity.py

```diff
@@ -170,57 +170,79 @@
+# :math:`p_{k,n_{\max}}` can be calculated similarly or by summing over its constituent orbit probabilities.
+#
+# Built-in functions :func:`~.feature_vector_orbits` and :func:`~.feature_vector_events`
+# can be used to get exact feature vectors. These functions
+# use a keyword argument ``samples`` to signal producing either exact or approximate probabilities.
+# ``samples`` is set to ``None`` to get exact feature vector by default. To use Monte Carlo estimation,
+# ``samples`` can be set to the number of samples desired to be used in the estimation, as shown later.
+# For example, to get the exact event probabilities in the feature vector example
+# :math:`f_{\mathbf{k} = (2, 4, 6), n_{\max}=2}` seen previously, we use:
+
+print(similarity.feature_vector_events(nx.Graph(m0_a), [2, 4, 6], 2, 6))
+
+##############################################################################
+# Although they are precise, exact calculations for large matrices can be tough to calculate. Additionally,
+# what makes calculating :math:`p_{k, n_{\max}}` really challenging is the number of samples the
+# corresponding event contains. For example, the 6-photon event over 17 modes :math:`E_{k=6, n_{\max}=2}`
 # contains the following number of samples :
 
 print(similarity.event_cardinality(6, 2, 17))
 
 ##############################################################################
-# To avoid calculating a large number of sample probabilities, an alternative is to perform a
-# Monte Carlo approximation. Here, samples within an event are selected uniformly at random and
-# their resultant probabilities are calculated. If :math:`N` samples :math:`\{S_{1}, S_{2},
-# \ldots , S_{N}\}` are generated, then the event probability can be approximated as
+# To avoid calculating a large number of sample probabilities, an alternative is to perform
+# Monte Carlo estimation. Here, samples within an orbit or event are selected uniformly
+# at random and their resultant probabilities are calculated. For example, for an event
+# :math:`E_{k, n_{\max}}`, if :math:`N` samples :math:`\{S_{1}, S_{2}, \ldots , S_{N}\}`
+# are generated, then the event probability can be approximated as
 #
 # .. math::
 #     p(E_{k, n_{\max}}) \approx \frac{1}{N}\sum_{i=1}^N p(S_i) |E_{k, n_{\max}}|,
 #
 # with :math:`|E_{k, n_{\max}}|` denoting the cardinality of the event.
 #
-# This method can be accessed using the :func:`~.prob_event_mc` function. The 4-photon event is
-# approximated as:
+# This method can be accessed using the :func:`~.feature_vector_events` function
+# with ``samples`` set to the number of samples desired to be used in the estimation.
+# For example, to get MC-estimated probabilities for our example feature vector
+# :math:`f_{\mathbf{k} = (2, 4, 6), n_{\max}=2}`, we use:
 
-print(similarity.prob_event_mc(nx.Graph(m0_a), 4, max_count_per_mode=2, n_mean=6))
+print(similarity.feature_vector_events(nx.Graph(m0_a), [2, 4, 6], 2, 6, samples=1000))

```

Comment:
I just tracked down why the tutorials take so long to build and it is this line - expecially for the 6 total photon number case this takes a long time. I'd recommend we work a way to instead have `prob_event_exact` for the 2 or 4 photon number case.